README.md

```diff
@@ -3,20 +3,84 @@ Terminal-based NetFlow/sFlow/IPFIX flow monitoring for OpenvSwitch
 
 Implemented using [tview](https://github.com/rivo/tview) (for graphics) and [goflow2](https://github.com/netsampler/goflow2) (for IPFIX collection).
 
+** Important Note: the word "flow" here means an IPFIX Flow Record not an OpenFlow flow or a datapath flow. For a more verbose definition, [please read the RFC](https://datatracker.ietf.org/doc/html/rfc7011#section-2)
+
 
 ## Building
 
     make
 
 ## Usage
 
-### Locally
-Start the ovn-flowmon daemon
+### Built-in OvS Exporter configuration
+There is a built-in OvS configuration module. If there is an OvS instance running locally, start the Flow Monitor using the "-ovsdb" option:
+
+    ./build/ovs-flowmon -ovsdb unix:/var/run/openvswitch/db.sock
+
+
+In the UI, you'll see a button to Configure IPFIX Exporter. You should use that button to configure OvS exporter **after** you have started the collector.
+
+If you have accidentally configured the exporter before starting the collector or you want to reconfigure the collector (e.g: to change the sampling rate), just you can re-configure as many times as you want.
+
+
+### Manual configuration
+If you are using an exporter other than OvS or it is not trivial how the exporter will access the collector (e.g. it's behind a router of some kind), you can start the Flow Monitor and manually configure the exporter.
+
+In that case, start the ovs-flowmon daemon without any option
+
+### Manual configuration
```

Comment:
@amorenoz duplicate "Manual configuration" section, I think you pasted the section twice

Reply:
Oops! :)
Thanks @mrVectorz